packages/core-packages-documentation-generator/src/sdk-index-link-client.ts

```diff
@@ -0,0 +1,48 @@
+import { isAbsolute, join, relative, resolve, sep } from "path";
+import { BindOption } from "typedoc";
+import { Component, RendererComponent } from "typedoc/dist/lib/output/components";
+import { PageEvent } from "typedoc/dist/lib/output/events";
+
+const PROJECT_ROOT = join(__dirname, "..", "..", "..", "..");
+@Component({ name: "SdkIndexLinkClientPlugin" })
+export class SdkIndexLinkClientPlugin extends RendererComponent {
+  @BindOption("out")
+  readonly out!: string;
+  /**
+   * The path pattern denotes the location of individual service client doc.
+   * "{{CLIENT}}" will be replaced with the client name.
+   * For example: `path/{{CLIENT}}/docs` will target s3 docs at `path/client-s3/docs`
+   */
+  @BindOption("clientDocs")
+  readonly clientDocs!: string;
+
+  initialize() {
+    this.listenTo(this.owner, {
+      [PageEvent.BEGIN]: this.onPageBegin,
+    });
+  }
+
+  onPageBegin(page: PageEvent) {
+    const out = isAbsolute(this.out) ? this.out : resolve(PROJECT_ROOT, this.out);
+    const clientDocs = isAbsolute(this.clientDocs) ? this.clientDocs : resolve(PROJECT_ROOT, this.clientDocs);
+    // Get relative path from core packages doc to clients' doc
```

Comment:
nit: period and a newline.
```suggestion

    // Get relative path from core packages doc to clients' doc.
```